### PR TITLE
Dapr/ch9 observability

### PR DESCRIPTION
**Pull request by EdwinVW**
Chapter 9 - Observability

### CODE REVIEW

**Comment by EdwinVW** [comment]
This is the Dapr for .NET Devs book chapter 9 about Observability.
@nishanil and @IEvangelist, please review. Thanks!

File: docs/architecture/dapr-for-net-developers/observability.md [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+| dapr_injector_sidecar_injection/requests_total | System | The total number of sidecar injection requests received by the Dapr Sidecar-Injector service. |
+| dapr_placement_runtimes_total                  | System | The total number of hosts reported to the Dapr Placement service. |
+| dapr_sentry_cert_sign_request_received_total   | System | The number of certificate signing requests (CRSs) received by the Dapr Sentry service. |
+| dapr_runtime_component_loaded      | Runtime | The number of successfully loaded Dapr components.           |
+| dapr_grpc_io_server_completed_rpcs | Runtime | Count of gRPC calls by method and status.                    |
+| dapr_http_server_request_count     | Runtime | Number of HTTP requests started in an HTTP server.           |
+| dapr_http/client/sent_bytes        | Runtime | Total bytes sent in request body (not including headers) by an HTTP client. |
+
+See the [Dapr metrics documentation](https://docs.dapr.io/developing-applications/building-blocks/observability/metrics/) for a complete list of available metrics.
```
Comment (IEvangelist) [inline docs/architecture/dapr-for-net-developers/observability.md:293]:
```suggestion
For more information on available metrics, see the [Dapr metrics documentation](https://docs.dapr.io/developing-applications/building-blocks/observability/metrics).
```

File: docs/architecture/dapr-for-net-developers/observability.md [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+Logging provides insight into what is happening with a service at runtime. When running an application, Dapr automatically emits log entries from Dapr sidecars and Dapr system services. However, logging entries instrumented in your application code **aren't** automatically included. To emit logging from application code, you can import a specific SDK like [OpenTelemetry SDK for .NET](https://opentelemetry.io/docs/net/). Logging application code is covered later in this chapter in the section *Using the Dapr .NET SDK*.  
+
+#### Log entry structure
+
+Dapr emits structured logging. Each log entry has the following format:
+
+| Field    | Description                                          | Example                             |
+| -------- | ---------------------------------------------------- | ----------------------------------- |
+| time     | ISO8601 formatted timestamp                          | `2021-01-10T14:19:31.000Z`          |
+| level    | Level of the entry (debug \| info \| warn  \| error) | `info`                              |
```
Comment (IEvangelist) [inline docs/architecture/dapr-for-net-developers/observability.md:335]:
```suggestion
| level    | Level of the entry (`debug` \| `info` \| `warn`  \| `error`) | `info`                              |
```

**Review by IEvangelist — CHANGES_REQUESTED** [review]
A few suggestions, tag me when you update this and I'll review it again.

**Comment by EdwinVW** [comment]
> A few suggestions, tag me when you update this and I'll review it again.

@IEvangelist Great suggestions! Applied them all.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**